app/src/main/res/values/strings.xml

```diff
@@ -626,7 +626,7 @@
 
     <string name="scan_first_product">Scan First Product</string>
 
-
+    <string name="preference_delete_search_history">Successfully deleted</string>
```

Comment:
Can you make this more specific? "Successfully deleted history" or something?

Reply:
yep.Thanks for the review.Changed it .Check it out now.

Reply:
@Karljoones I didn't made any extra changes ..but build is failing.

Reply:
It's possible that another merge has made it fail